component-classes/shortcuts.js

```diff
@@ -1,17 +1,18 @@
 import { button } from './index.js';
 
 export const buttons = {
-  'button': button.buttonSecondary,
-  'button--secondary': button.buttonSecondary,
-  'button--default': button.buttonSecondary,
-  'button--small': button.buttonSmall,
+  'button': button.secondary,
+  'button--secondary': button.secondary,
+  'button--default': button.secondary,
+  'button--small': button.small,
   'button--link': button.buttonLink,
-  'button--primary': button.buttonPrimary,
-  'button--cta': button.buttonPrimary,
-  'button--pill': button.buttonPill,
+  'button--primary': button.primary,
+  'button--cta': button.primary,
+  'button--pill': button.pill,
   'button--utility': button.buttonUtility,
   'button--utility-flat': button.buttonUtilityFlat,
   'button--destructive': button.buttonDestructive,
   'button--destructive-flat': button.buttonDestructiveFlat,
-  'button--flat': button.buttonFlat,
+  'button--flat': button.quiet,
+  'button--quiet': button.quiet,
```

Comment:
Did we have `button--quiet` in Fabric that we need to add a shortcut for it?

Reply:
I was going to say yes as I did add it on purpose here. But I've searched now in finn and couldn't find any usages of it. So I don't know why I added it 🤔 I'll remove it

Reply:
https://github.com/warp-ds/css/pull/44/commits/128a08be9656a147be598407303b6c4b31030416